### PR TITLE
Detached (Remote)StreamEnvironment execution

### PR DESCRIPTION
**Pull request by nragon**
Detached execution option from api

### CODE REVIEW

File: flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java [hunk 1]
```diff
@@ -1498,10 +1499,12 @@ public JobExecutionResult execute() throws Exception {
 	 *
 	 * @param jobName
 	 * 		Desired name of the job
+	 * 	@param detached
+	 * 		Whether to run the job in detached mode
 	 * @return The result of the job execution, containing elapsed time and accumulators.
 	 * @throws Exception which occurs during job execution.
 	 */
-	public abstract JobExecutionResult execute(String jobName) throws Exception;
+	public abstract JobExecutionResult execute(String jobName, boolean detached) throws Exception;
```
Comment (zentol) [inline flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java]: this is a public method that you mustn't change; you have to add a completely new method.

**Comment by nragon** [comment]
I guess for backward compatibility this last commig would do.

File: flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/RemoteStreamEnvironment.java [hunk 2]
```diff
@@ -25,229 +25,205 @@
...
 
 @Public
 public class RemoteStreamEnvironment extends StreamExecutionEnvironment {
-	
-	private static final Logger LOG = LoggerFactory.getLogger(RemoteStreamEnvironment.class);
-
-	/** The hostname of the JobManager */
-	private final String host;
-
-	/** The port of the JobManager main actor system */
```
Comment (zentol) [inline flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/RemoteStreamEnvironment.java]: please revert the auto-formatting here.

**Comment by bachrc** [comment]
Why isn't it implemented yet ? Expect the conflicting files, it's still not possible to launch a flink job in a detached mode.

**Pull request closed without merging**